councilmatic_core/views.py

```diff
@@ -134,8 +134,7 @@ class CouncilMembersView(ListView):
     context_object_name = 'posts'
     
     def get_queryset(self):
-        return Organization.objects.get(ocd_id=settings.OCD_CITY_COUNCIL_ID).posts.all()
-        # return Post.objects.filter(organization__ocd_id=settings.OCD_CITY_COUNCIL_ID)
+        return Post.objects.all()
```

Comment:
Is it proper to assume that only posts from the city will be in the database? If so, realized there is no need to filter, right?